test/functionalTest/cases/2750_common_metrics/2750_incoming_errors.test

```diff
@@ -90,12 +90,13 @@ brokerStart CB
 # 05. Send a request with an Invalid Service Path
 # 06. Send a request with an Invalid Content-Type
 # 07. Send a request with invalid value for URI param /options/
-# 08. Send a request with Content-Length > PAYLOAD_MAX_SIZE
+# 08. Send a request with a component-name too long in ServicePath
```

Comment:
Yeah, this is an older change. The ToC wasn't changed when it was supposed to be